users/serializers.py

```diff
@@ -31,6 +36,9 @@
 USERNAME_RE_PARTIAL = r"[\w ._+-]+"
 USERNAME_RE = re.compile(rf"(?P<username>{USERNAME_RE_PARTIAL})")
 USERNAME_ERROR_MSG = "Username can only contain letters, numbers, spaces, and the following characters: _+-"
+USERNAME_ALREADY_EXISTS_MSG = (
```

Comment:
Oh, you've already created one, I have a comment above where I mentioned creating a constant, maybe this is something we can reuse.